content/applications/inventory_and_mrp/purchase/advanced/procurement_expenses_report.rst

```diff
@@ -0,0 +1,113 @@
+   changed by selecting either :guilabel:`Requests for Quotation` or :guilabel:`Purchase Orders`
+   under the :guilabel:`Filters` column.
+
+Under the :guilabel:`Filters` column, select a date range to use for comparison. The report can be
+filtered by either :guilabel:`Order Date` or :guilabel:`Confirmation Date`. Choose one from the
+list, and click the |caret| to specify the date range, either by month, quarter, or year.
+
+Next, under the :guilabel:`Group by` column, select :guilabel:`Vendor`. Then, select
+:guilabel:`Product Category`, which is also located in the :guilabel:`Group By` column.
+
+.. note::
+   The selections under the :guilabel:`Group By` heading can be altered depending on the needs of
+   the individual company. For example, selecting :guilabel:`Product` instead of :guilabel:`Product
+   Category` provides a more in depth look at the performance of specific items, instead of an
+   entire category.
+
+Next, make a selection under the :guilabel:`Comparison` heading. These options are only available
+after the date range is selected under the :guilabel:`Filters` column, and vary based on that range.
+:guilabel:`Previous Period` adds a comparison to the previous period, such as the last month or
+quarter. :guilabel:`Previous Year` compares the same time period from the previous year.
+
+.. note::
+   While multiple time-based filters can be added at once, only one comparison can be selected at a
+   time.
+
+.. image:: procurement_expenses_report/filters-groups.png
+   :align: center
+   :alt: The drop-down menu of filters, group by and comparison options for the procurement expenses
+         report.
+
+Add measures
+------------
+
+After selecting the :guilabel:`Filters`, :guilabel:`Group by`, and :guilabel:`Comparison` settings,
+click out of the drop-down menu.
+
+By default, the report displays with the following measures: :guilabel:`Order`, :guilabel:`Total`,
+:guilabel:`Untaxed Total`, and :guilabel:`Count`. Click :guilabel:`Measures` at the top-left to open
+the drop-down list of available measures.
+
+Click on the following measures to include them in the report:
+
+- :guilabel:`Total` and :guilabel:`Untaxed Total`: can include one or both measures. These are
+  included for overall spending analysis.
+- :guilabel:`Average Cost`: included to evaluate cost efficiency.
+- :guilabel:`Days to Confirm` and :guilabel:`Days to Receive`: used to assess supplier performance.
+- :guilabel:`Qty Ordered` and :guilabel:`Qty Received`: used to understand order efficiency.
+- :guilabel:`Qty Billed` and :guilabel:`Qty to be Billed`: used to track order accuracy.
+
+.. tip::
+   Additional measures can be included in the report, if desired, to provide additional insights.
+   For example, :guilabel:`Gross Weight` and :guilabel:`Volume` may be included for further
+   logistics and handling analysis.
+
+After selecting all necessary measures, click out of the drop-down menu.
+
+View results
+============
+
```

Comment:
Just my impression, but I was expecting to see an example procurement expense report here, rather than just instructions to add it into a spreadsheet. I think it would be helpful for readers to get a visual of what an ideal expense report looks like. Was there a reason the image wasn’t included? Just curious!

![image](https://github.com/odoo/documentation/assets/23611711/a1bcace2-7402-4c9f-928e-d26a9b8594b5)

Reply:
Because of the way the report generates as a pivot, it is such a massive screenshot that it keeps showing up pretty blurry, which didn't look too great. But  you are right, it really should be included. I will try again. :+1: